test/black-box/Address.test.ts

```diff
@@ -0,0 +1,127 @@
+// import { api } from './lib/api';
+// import { DatabaseResetCommand } from '../../src/console/DatabaseResetCommand';
```

Comment:
uncomment, these are the minimum tests that should succeed for each model and they will run out of the box after test data is defined and need just few changes if there are relations.